### PR TITLE
feat: document mdx components in storybook

### LINKED ISSUE #644: Add Caption MDX component to storybook

### LINKED ISSUE #645: Add PageDescription MDX component to storybook

### LINKED ISSUE #646: Add Title MDX component to storybook

### LINKED ISSUE #648: Add ArtDirection MDX component to storybook

### LINKED ISSUE #650: Add GifPlayer MDX component to storybook

### LINKED ISSUE #651: Add InlineNotification MDX component to storybook

### LINKED ISSUE #652: Add Tabs, Tab MDX components to storybook

### LINKED ISSUE #653: Add Video MDX component to storybook

### LINKED ISSUE #654: Add ResourceCard MDX component to storybook

### LINKED ISSUE #655: Add Aside MDX component to storybook

### LINKED ISSUE #656: Add Profile MDX component to storybook

### PR DESCRIPTION
**Pull request by alisonjoseph**
Closes #653, Closes #644, Closes #651, Closes #655, Closes #646, Closes #645, Closes #654, Closes #656, Closes #650, Closes #648, Closes #652, Closes #657

#### Add the following components to Storybook

- Caption
- Title
- PageDescription
- ResourceCard
- Aside,
- InlineNotification
- Profile
- GifPlayer
- ArtDirection
- Video
- Tabs
- ArticleCard

#### Testing / reviewing

http://localhost:6006/

### TODO
- [x] Get next/js image working in storybook
- [x] ArtDirection description / proptypes not displaying
- [x] Video preview is cropping example

### CODE REVIEW

**Review by andreancardona — DISMISSED** [review]

File: services/web-app/components/art-direction/art-direction1.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
+/*
```
Comment (francinelucca) [inline services/web-app/components/art-direction/art-direction1.js:1]: do we need an art-direction.js and an art-direction1.js?

File: services/web-app/stories/mdx/dondont.stories.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
+/*
```
Comment (francinelucca) [inline services/web-app/stories/mdx/dondont.stories.js:1]: rename:

services/web-app/stories/mdx/dondont.stories.js -> services/web-app/stories/mdx/dodont.stories.js

**Review by francinelucca — CHANGES_REQUESTED** [review]
any reason for moving this file?:
 services/web-app/scripts/create-local-certificates.sh  -> services/web-app/stories/scripts/create-local-certificates.sh

File: services/web-app/stories/mdx/art-direction.stories.js [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+import Image from 'next/image'
+
+import ArtDirection from '@/components/art-direction'
+
+const stories = {
+  title: 'MDX/ArtDirection',
+  component: ArtDirection
```
Comment (francinelucca) [inline services/web-app/stories/mdx/art-direction.stories.js:13]:
```suggestion
  component: ArtDirection,
    argTypes: {
    children: {
      control: false
    }
  }
```

File: services/web-app/stories/mdx/article-card.stories.js [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+ * LICENSE file in the root directory of this source tree.
+ */
+import { Column, Grid } from '@carbon/react'
+import Image from 'next/image'
+
+import ArticleCard from '@/components/article-card'
+
+const stories = {
+  title: 'MDX/ArticleCard',
+  component: ArticleCard
```
Comment (francinelucca) [inline services/web-app/stories/mdx/article-card.stories.js:14]:
```suggestion
  component: ArticleCard,
     argTypes: {
    children: {
      control: false
    },
    className: {
    control: false
    
    }
    }
```

File: services/web-app/stories/mdx/aside.stories.js [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+import { Column, Grid } from '@carbon/react'
+
+import Aside from '@/components/aside'
+
+const stories = {
+  title: 'MDX/Aside',
+  component: Aside
```
Comment (francinelucca) [inline services/web-app/stories/mdx/aside.stories.js:13]:
```suggestion
  component: Aside,
     argTypes: {
    children: {
      control: false
    },
    className: {
    control: false
    
    }
    }
```

File: services/web-app/stories/mdx/dondont.stories.js [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+ * LICENSE file in the root directory of this source tree.
+ */
+import Image from 'next/image'
+
+import { DoDont, DoDontRow } from '@/components/do-dont'
+
+const stories = {
+  title: 'MDX/DoDont',
+  component: DoDontRow,
+  subcomponents: { DoDont }
```
Comment (francinelucca) [inline services/web-app/stories/mdx/dondont.stories.js:14]:
```suggestion
  subcomponents: { DoDont },
     argTypes: {
    children: {
      control: false
    },
    className: {
    control: false
    
    }
    }
```

File: services/web-app/stories/mdx/gif-player.stories.js [hunk 7]
```diff
@@ -0,0 +1,26 @@
...
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+import Image from 'next/image'
+
+import GifPlayer from '@/components/gif-player'
+
+const stories = {
+  title: 'MDX/GifPlayer',
+  component: GifPlayer
```
Comment (francinelucca) [inline services/web-app/stories/mdx/gif-player.stories.js:13]:
```suggestion
  component: GifPlayer,
     argTypes: {
    children: {
      control: false
    },
    className: {
    control: false
    
    }
    }
```

File: services/web-app/stories/mdx/inline-notification.stories.js [hunk 8]
```diff
@@ -0,0 +1,46 @@
...
+ * Copyright IBM Corp. 2022, 2022
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+import InlineNotification from '@/components/inline-notification'
+
+const stories = {
+  title: 'MDX/InlineNotification',
+  component: InlineNotification
```
Comment (francinelucca) [inline services/web-app/stories/mdx/inline-notification.stories.js:11]:
```suggestion
  component: InlineNotification,
     argTypes: {
    children: {
      control: false
    },
    className: {
    control: false
    
    }
    }
```

File: services/web-app/stories/mdx/page-description.stories.js [hunk 9]
```diff
@@ -0,0 +1,25 @@
...
+ * Copyright IBM Corp. 2022, 2022
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+import PageDescription from '@/components/page-description'
+
+const stories = {
+  title: 'MDX/PageDescription',
+  component: PageDescription
```
Comment (francinelucca) [inline services/web-app/stories/mdx/page-description.stories.js:11]:
```suggestion
  component: PageDescription,
     argTypes: {
    children: {
      control: false
    },
    className: {
    control: false
    
    }
    }
```

File: services/web-app/stories/mdx/profile.stories.js [hunk 10]
```diff
@@ -0,0 +1,31 @@
...
+ * LICENSE file in the root directory of this source tree.
+ */
+import { Grid } from '@carbon/react'
+import Image from 'next/image'
+
+import Profile from '@/components/profile'
+
+const stories = {
+  title: 'MDX/Profile',
+  component: Profile
```
Comment (francinelucca) [inline services/web-app/stories/mdx/profile.stories.js:14]:
```suggestion
  component: Profile,
     argTypes: {
    children: {
      control: false
    }
    }
```

File: services/web-app/stories/mdx/resource-card.stories.js [hunk 11]
```diff
@@ -0,0 +1,68 @@
...
+ * LICENSE file in the root directory of this source tree.
+ */
+import { Column, Grid } from '@carbon/react'
+import Image from 'next/image'
+
+import ResourceCard from '@/components/resource-card'
+
+const stories = {
+  title: 'MDX/ResourceCard',
+  component: ResourceCard
```
Comment (francinelucca) [inline services/web-app/stories/mdx/resource-card.stories.js:14]:
```suggestion
  component: ResourceCard,
     argTypes: {
    children: {
      control: false
    },
    className: {
    control: false
    
    }
    }
```

File: services/web-app/stories/mdx/tabs.stories.js [hunk 12]
```diff
@@ -0,0 +1,41 @@
...
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+import { Tab, Tabs } from '@/components/tabs'
+
+const stories = {
+  title: 'MDX/Tabs',
+  component: Tabs,
+  subcomponents: { Tab }
```
Comment (francinelucca) [inline services/web-app/stories/mdx/tabs.stories.js:12]:
```suggestion
  subcomponents: { Tab },,
     argTypes: {
    children: {
      control: false
    }
    }
```

File: services/web-app/stories/mdx/title.stories.js [hunk 13]
```diff
@@ -0,0 +1,19 @@
...
+ * Copyright IBM Corp. 2022, 2022
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+import Title from '@/components/title'
+
+const stories = {
+  title: 'MDX/Title',
+  component: Title
```
Comment (francinelucca) [inline services/web-app/stories/mdx/title.stories.js:11]:
```suggestion
  component: Title,,
     argTypes: {
    children: {
      control: false
    },
    className: {
    control: false
    
    }
    }
```

File: services/web-app/stories/mdx/video.stories.js [hunk 14]
```diff
@@ -0,0 +1,37 @@
...
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+import Video from '@/components/video'
+import localPoster from '@/pages/mdx/video/local-poster.jpeg'
+import localVideo from '@/pages/mdx/video/local-video.mp4'
+
+const stories = {
+  title: 'MDX/Video',
+  component: Video
```
Comment (francinelucca) [inline services/web-app/stories/mdx/video.stories.js:13]:
```suggestion
  component: Video,
     argTypes: {
    children: {
      control: false
    },
    src: {
    control: false
    },
    poster: {
    control: false
    }
    }
```

**Review by francinelucca — COMMENTED** [review]
Suggestions for controls we might not want to show on SB

**Review by francinelucca — APPROVED** [review]
👨‍🍳 💋

**Pull request merged**